proto/atomone/gov/v1/gov.proto

```diff
@@ -349,4 +349,22 @@ message Params {
 
   // Minimum proportion of No Votes for a proposal deposit to be burnt.
   string burn_deposit_no_threshold = 25 [(cosmos_proto.scalar) = "cosmos.Dec"];
+
+  // Maximum achievable quorum
+  string max_quorum = 26 [(cosmos_proto.scalar) = "cosmos.Dec"];
+
+  // Minimum achievable quorum
+  string min_quorum = 27 [(cosmos_proto.scalar) = "cosmos.Dec"];
+
+  // Maximum achievable quorum for constitution amendment proposals
+  string max_constitution_amendment_quorum = 28 [(cosmos_proto.scalar) = "cosmos.Dec"];
+
+  // Minimum achievable quorum for constitution amendment proposals
+  string min_constitution_amendment_quorum = 29 [(cosmos_proto.scalar) = "cosmos.Dec"];
+
+  // Maximum achievable quorum for law proposals
+  string max_law_quorum = 30 [(cosmos_proto.scalar) = "cosmos.Dec"];
+
+  // Minimum achievable quorum for law proposals
+  string min_law_quorum = 31 [(cosmos_proto.scalar) = "cosmos.Dec"];
```

Comment:
wouldn't it be better to have a struct with both caps inside for each quorum? (like for dynamic deposit) instead of separate min/max params?

Reply:
Yes, what about embedding the threshold as well ?

Reply:
I created an issue to track the remaining work https://github.com/atomone-hub/atomone/issues/145